submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,79 @@
+from sprint_b.helpers import FileManager, Vocabulary, EpithetGenerator
+from flask import json
+import os
+import json
+import shutil
+
+"""     use pytest      """
+
+path = "../../resources/data.json"
+sad_path = ".fake./.path./resources123/data321.json"
+
+with open(os.path.abspath(path)) as json_data:
+    data = json.load(json_data)
+
+def test_get_extension():
+    assert FileManager.get_extension(path) == "json"
+
+
+def test_read_json():
+    assert FileManager.read_json(path) == data
+
+
+def test_from_file():
+    assert Vocabulary.from_file(path) == (data, data.keys())
+
+
+def test_from_json():
+    assert Vocabulary.from_json(path) == (data, data.keys())
+
+
+def test_strategies():
+    assert Vocabulary.strategies("json") == Vocabulary.from_json
+
+def test_happy_path():
+    """Tests that the first word in the generated epithet is Thou,
```

Comment:
"Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description."
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

The summary line should also end with a period.